staging/src/k8s.io/apiserver/pkg/authentication/authenticator/helpers.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package authenticator
+
+// APIAudiences is an Audiences that contains only the default audience
+// identifier of the Kubernetes API server.
+var APIAudiences = Audiences{""}
+
+// Audiences is a container for the Audiences of a token.
+type Audiences []string
```

Comment:
Should have a helper method like `HasKubeAPIServer` to avoid all the `aud.Has("")`